packages/x-components/src/components/modals/__tests__/x-modal.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { mount, Wrapper } from '@vue/test-utils';
+import Vue from 'vue';
+import { getDataTestSelector, installNewXPlugin } from '../../../__tests__/utils';
+import XModal from '../x-modal.vue';
+import { PropsWithType } from '../../../utils/types';
+import { XEventsTypes } from '../../../wiring/events.types';
+
+/**
+ * Renders a {@link XModal} component with the provided options and offers an API to easily
+ * test it.
+ *
+ * @param options - The options to render the component with.
+ * @returns An API to test the component.
+ */
+function renderXModal({ template = '<XModal/>' }: RenderXModalOptions = {}): RenderXModalAPI {
+  const [, localVue] = installNewXPlugin();
+  const parent = document.createElement('div');
+  document.body.appendChild(parent);
+
+  const wrapper = mount(
+    {
+      template,
+      components: { XModal }
+    },
+    {
+      attachTo: parent,
```

Comment:
Why is this necessary? I removed it and the tests are passing in any case.
Add a little comment explaining why.

Also, according to the doc, you can use the body directly, and a div will be created automatically:
![image](https://user-images.githubusercontent.com/5759712/179922204-c726aa07-164a-4117-ac1e-c710d5eb6e89.png)